package.json

```diff
@@ -95,6 +93,10 @@
     "node": "4.x || 5.x || 6.x || 7.x",
     "npm": "2.x || 3.x || 4.x"
   },
+  "dependencies": {
```

Comment:
D'oh! Habit. I'll undo _this_ change and update the `packages/*/package.json` files instead 😄